ga4gh/server/network/__init__.py

```diff
@@ -0,0 +1,76 @@
+"""
+Provides methods to initialize the server's peer to peer connections.
+
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import requests
+
+from ga4gh.server import datamodel, exceptions
```

Comment:
Another consideration is searching through the code... it is much easier to grep for 'import ga4gh.server.datamodel.variants' to find all the places it is imported than... I'm not even sure what regex one would write for the alternatives.